src/v0/destinations/iterable/util.test.js

```diff
@@ -799,4 +800,155 @@ describe('iterable utils test', () => {
       );
     });
   });
+  describe('checkIfEventIsAbortableAndExtractErrorMessage', () => {
```

Comment:
Can we add test-case with multiple path matches for same email(for example) as well ?